receiver/hostmetricsreceiver/internal/scraper/processesscraper/processes_scraper_linux.go

```diff
@@ -25,9 +25,8 @@ import (
 
 const unixSystemSpecificMetricsLen = 1
 
-func appendUnixSystemSpecificProcessesMetrics(metrics pdata.MetricSlice, startIndex int, now pdata.Timestamp, misc *load.MiscStat) error {
+func appendUnixSystemSpecificProcessesMetrics(metrics pdata.MetricSlice, startIndex int, now pdata.Timestamp, misc *load.MiscStat) {
 	initializeProcessesCreatedMetric(metrics.At(startIndex), now, misc)
-	return nil
```

Comment:
I think this returns error on other os-es. So may not be ok to remove it?

Reply:
It doesn't return an error anywhere currently (checked with `git grep -A 3 'func appendUnixSystemSpecificProcessesMetrics'` ), but it might in the future so I added the `return` back and checked for the error instead.